From 8eb56ad7fe14e19ed24ec59531fd5f26af1af08e Mon Sep 17 00:00:00 2001
From: DX37 <asusx2@mail.ru>
Date: Wed, 15 Aug 2018 11:29:54 +0700
Subject: [PATCH] rename async vars

Since Python 3.7, async and await is keywords now.
Renamed async variables for compability.
---
 ydcmd.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ydcmd.py b/ydcmd.py
index 1f18240..49d84fd 100755
--- a/ydcmd.py
+++ b/ydcmd.py
@@ -546,7 +546,7 @@ def __init__(self, config):
         self.quiet            = self._bool(config["quiet"])
         self.debug            = self._bool(config["debug"]) and not self.quiet
         self.verbose          = (self._bool(config["verbose"]) or self.debug) and not self.quiet
-        self.async            = self._bool(config["async"])
+        self.async_d          = self._bool(config["async"])
         self.rsync            = self._bool(config["rsync"])
         self.recursion        = not self._bool(config["no-recursion"])
         self.no_recursion_tag = str(config["no-recursion-tag"])
@@ -973,7 +973,7 @@ def yd_wait(options, link):
         options (ydOptions) -- Опции приложения
         link    (dict)      -- Ответ API на запрос операции
     """
-    if options.async or not ("href" in link and "method" in link):
+    if options.async_d or not ("href" in link and "method" in link):
         return
 
     url    = link["href"]
